python-threatexchange/threatexchange/hashing/pdq_utils.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+BITS_IN_PDQ = 256
+
+
+def simple_distance_binary(bin_a, bin_b):
```

Comment:
Nit: Since this is a file of utility methods, some basic docstrings for each method may be helpful when trying to use them in an IDE.